components/components-style.css

```diff
@@ -11,34 +11,37 @@
 @import url("./lists.css");
 @import url("./snackbar.css");
 @import url("./grid.css");
-
+@import url("./dark-mode.css");
 
 * {
   font-family: "Readex Pro", sans-serif;
   transition: all ease 0.5s;
   box-sizing: border-box;
   margin: 0;
   padding: 0;
+    /* background-color: var(--secondary-color);
+  color: var(--font-color); */
```

Comment:
minor suggestion: commented code could be removed

Reply:
Thank you for the suggestion, But I've kept it intentionally for reference.